c8run/go.mod

```diff
@@ -1,3 +1,4 @@
-module c8run
+module github.com/camunda/camunda/c8run
 
+replace github.com/camunda/camunda/c8run => ./
```

Comment:
Not sure why the replace directive is needed? Usually that would only be required if you "replace" a public module with an external one.

Reply:
Already removed it in a later commit. I realized it wasn't necessary.

I thought what would happen is if you import github.com/camunda/camunda/c8run then it would  do a git clone on main and use that for your library, and then it'd be a mismatch of which versions of the library you're using.

Reply:
https://github.com/camunda/camunda/pull/24040/commits/ad6c501685ea273d36a8d96562c78be0afc236be